javascript/javascript3/week1/json-file.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "ABBA",
+  "origin": ["stokholm", "Sweden"],
```

This may not be the best way to structure an `origin` field. 

If you want the origin country and city be separated, you can put them in an origin object. 

For example:

```suggestion
  "origin":{
     "city": "stokholm", 
     "country": "Sweden"
  },
```